packages/core/package.json

```
@@ -3,32 +3,35 @@
   "version": "2.0.0-alpha.2",
   "description": "A simple React component capable of building HTML forms out of a JSON schema.",
   "scripts": {
-    "build:lib": "rimraf lib && cross-env NODE_ENV=production babel -d lib/ src/",
-    "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config webpack.config.dist.js",
-    "cs-check": "prettier -l \"{playground,src,test}/**/*.js\"",
-    "cs-format": "prettier \"{playground,src,test}/**/*.js\" --write",
-    "dist": "npm run build:lib && npm run build:dist",
-    "lint": "eslint src test playground",
+    "build": "npm run dist:build:umd && npm run build:cjs && npm run build:es && npm run dist:build:es:lib",
+    "build:cjs": "cross-env NODE_ENV=production BABEL_ENV=cjs babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./dist/cjs",
+    "build:es": "cross-env NODE_ENV=production BABEL_ENV=es babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./dist/es",
+    "dist:build:es:lib": "cross-env NODE_ENV=production BABEL_ENV=es babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./lib",
+    "dist:build:umd": "rimraf dist && cross-env NODE_ENV=production BABEL_ENV=umd webpack --config webpack.config.dist.js",
+    "cs-check": "prettier -l \"{src,test}/**/*.js\"",
+    "cs-format": "prettier \"{src,test}/**/*.js\" --write",
+    "lint": "eslint src test",
     "precommit": "lint-staged",
-    "publish-to-npm": "npm run dist && npm publish",
-    "preversion": "npm run build:playground && npm run dist && npm run cs-check && npm run lint",
-    "start": "node devServer.js",
+    "publish-to-npm": "npm run build && npm publish",
+    "preversion": "npm run build && npm run cs-check && npm run lint",
+    "start": "concurrently \"npm:build:* -- --watch\"",
     "tdd": "cross-env NODE_ENV=test mocha --require @babel/register --watch --require ./test/setup-jsdom.js test/**/*_test.js",
     "test": "cross-env NODE_ENV=test mocha --require @babel/register --require ./test/setup-jsdom.js test/**/*_test.js",
     "test-coverage": "cross-env NODE_ENV=test nyc --reporter=lcov mocha --require @babel/register --require ./test/setup-jsdom.js test/**/*_test.js",
     "test-debug": "cross-env NODE_ENV=test mocha --require @babel/register --require ./test/setup-jsdom.js --debug-brk --inspect test/Form_test.js"
   },
   "lint-staged": {
-    "{playground,src,test}/**/*.js": [
+    "{src,test}/**/*.js": [
       "eslint --fix",
       "prettier --write",
       "git add"
     ]
   },
-  "main": "lib/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/es/index.js",
```

Probably want to add this instead of including it in the `files` array.
```suggestion
  "module": "dist/es/index.js",
  "typings": "index.d.ts"
```

---

Done